### PR TITLE
Added message memory for conversations

### PR DESCRIPTION
**Pull request by ppajersk**
Added replying to threads to compliment the first change
Added more dialog
Added issue retrieving from a git repo (env var)
Added some placeholder verification when requesting-
-issues from a repo (Will be changed)

### CODE REVIEW

File: wxt_cidrbot/Git_handler.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from github import Github
+
+
+class githandler:
+    def __init__(self):
+        logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+        self.logging = logging.getLogger()
+
+        # Connect to git, right now using no tokens
+        self.git_init = Github()
```
Comment (jlothian) [inline wxt_cidrbot/Git_handler.py:13]: a better name for `git_init` would be something like `git_api_handle` or `git_api` -

File: wxt_cidrbot/Git_handler.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        # List of git repos to pull from
+        if "GIT_REPO_LIST" in os.environ:
+            self.repos = os.getenv("GIT_REPO_LIST")
+        else:
+            logging.error("Environment variable GIT_REPO_LIST must be set")
+            sys.exit(1)
+
+    def scan_repos(self):
+        self.repos = self.repos.split(",")
```
Comment (jlothian) [inline wxt_cidrbot/Git_handler.py:23]: this line should be in __init__

**Review by jlothian — CHANGES_REQUESTED** [review]
General comment - for your module names, either use CamelCase or snake_case, but not both.  I think the python convention is generally snake case.

**Comment by ppajersk** [comment]
Changes should be made. I didn't snake case the class names, they are all 1 word and lowercased. I couldn't find any function names that where CamelCased

**Comment by jlothian** [comment]
> Necessary changes are be made. I didn't snake case the class names, they are all 1 word and lowercased. I couldn't find any function names that were CamelCased

Looks like you still need to rename the modules - Github_handler, Command_list, etc.

**Review by jlothian — APPROVED** [review]
LGTM

**Pull request merged**